documents/src/pages/elements/interactive-chart.md

```diff
@@ -494,7 +496,7 @@ chart.config = {
       },
       data: [
         { time: '2020-01-11', value: 20.31 },
-        { time: '2020-02-12', value: 30.27 },
+        { time: '2020-02-13', value: 30.27 },
```

Comment:
Wondering why do we need this change?

Reply:
It will help the developer to see the effect of the formatter on the seasonality chart, ensuring that duplicate months are not shown.